### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by AnthonyLaw**
- Add toggle theme (light / dark) on menu
- Default theme will be dark
- Replace the transaction type icon
- Replace the new symbol icon
- Replace font type
- minor adjustment on Header and Footer

Resolved: #902

### CODE REVIEW

File: src/components/ThemeToggle.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+</template>
+
+<script>
+import IconMoon from '../styles/img/moon.png';
+import IconSun from '../styles/img/sun.png';
+
+export default {
+	mounted() {
+		// get theme from localstorage
+		let theme = localStorage.getItem('theme');
```
Comment (OlegMakarenko) [inline src/components/ThemeToggle.vue:35]: I think we should not access localStorage directly from UI components. I would move it to store or service

File: src/components/menu/MobileMenu.vue [hunk 2]
```diff
@@ -89,14 +93,25 @@ export default {
...
 		getNameByKey(e) {
 			return this.$store.getters['ui/getNameByKey'](e);
+		},
+
+		iconUrl(icon) {
+			switch (icon) {
+			case 'IconStatistics':
+				return this.IconStatistics;
+			case 'IconMosaics':
+				return this.IconMosaics;
```
Comment (OlegMakarenko) [inline src/components/menu/MobileMenu.vue:103]: Why there is only `Statistic` and `Mosaic` icons?

File: src/components/menu/NavigationMenu.vue [hunk 3]
```diff
@@ -68,33 +70,36 @@ export default {
...
 	methods: {
 		getNameByKey(e) {
 			return this.$store.getters['ui/getNameByKey'](e);
+		},
+		iconUrl(icon) {
+			switch (icon) {
+			case 'IconStatistics':
+				return this.IconStatistics;
+			case 'IconMosaics':
+				return this.IconMosaics;
```
Comment (OlegMakarenko) [inline src/components/menu/NavigationMenu.vue:88]: Why there is only `Statistic` and `Mosaic` icons?

File: src/components/ThemeToggle.vue [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+			// Save theme in local storage
+			localStorage.setItem('theme', darkMode);
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+.toogleTheme {
```
Comment (OlegMakarenko) [inline src/components/ThemeToggle.vue:70]:
```suggestion
.toogle-theme {
```

And above

File: src/components/menu/MobileMenu.vue (hunk 2 above)
Comment (AnthonyLaw) [reply src/components/menu/MobileMenu.vue:103]: That 2 new icons from the design team.

File: src/components/ThemeToggle.vue (hunk 1 above)
Comment (AnthonyLaw) [reply src/components/ThemeToggle.vue:35]: The reason put into localstorage is to prevent theme reset.

**Review by OlegMakarenko — APPROVED** [review]

**Pull request merged**